osu.Game/Screens/Play/Player.cs

```diff
@@ -1224,11 +1224,7 @@ protected virtual Task ImportScore(Score score)
         /// </summary>
         /// <param name="score">The <see cref="ScoreInfo"/> to be displayed in the results screen.</param>
         /// <returns>The <see cref="ResultsScreen"/>.</returns>
-        protected virtual ResultsScreen CreateResults(ScoreInfo score) => new SoloResultsScreen(score)
-        {
-            AllowRetry = true,
-            ShowUserStatistics = true,
```

Comment:
Why are these removed from here? I would hope they could stay here. Otherwise this is going to take significant checking that every single derived implementation sets these correctly.

Reply:
I mean sure, but only `EditorPlayer` would not override this in the current form.